ansible/roles/real-postfix/files/postfix/pass-scripts.mit.edu

```diff
@@ -0,0 +1,3 @@
+# Match @scripts.mit.edu addresses (but don't change them)
+# This is chained with an LDAP lookup using pipemap
+/(.*)@scripts\.mit\.edu/ $1@scripts.mit.edu
```

Comment:
Again, should this be anchored? I guess in this case, assuming there's no implicit anchoring, it could actually cause weirdness if somebody registers like scripts.mit.edu.example.com...

Reply:
Should this use `([^+]*)` instead of `(.*)`? I mean, presumably we're never going to have a `foo+bar.scripts.mit.edu` vhost anyway, so it shouldn't matter...

Reply:
This should not exclude +; that is matching the local-part of the address, which can contain plusses.

Reply:
Hm, I'm not sure what I was thinking here. Sorry.

Reply:
Oh, maybe my concern was a worry that pass-scripts.mit.edu should match things without a suffix to avoid conflicting with pass-scripts.mit.edu-suffix, rather than matching everything, but I guess it works.